kpops/components/base_components/kubernetes_app.py

```diff
@@ -72,31 +72,26 @@ def namespace(self) -> str:
 
     def deploy(self, dry_run: bool) -> None:
         stdout = self.helm_wrapper.upgrade_install(
-            release_name=self.helm_release_name,
-            chart=self.get_helm_chart(),
-            dry_run=dry_run,
-            namespace=self.namespace,
-            values=self.to_helm_values(),
-            flags=HelmUpgradeInstallFlags(version=self.get_helm_chart_version()),
+            self.helm_release_name,
+            self.get_helm_chart(),
+            dry_run,
+            self.namespace,
+            self.to_helm_values(),
+            HelmUpgradeInstallFlags(version=self.get_helm_chart_version()),
         )
 
         if dry_run and self.helm_diff.config.enable:
             self.print_helm_diff(stdout)
 
+    # TODO: Separate destroy and clean
     def destroy(self, dry_run: bool, clean: bool, delete_outputs: bool) -> None:
-        try:
-            stdout = self.helm_wrapper.uninstall(
-                namespace=self.namespace,
-                release_name=self.helm_release_name,
-                dry_run=dry_run,
-            )
-            if dry_run and self.helm_diff.config.enable:
-                self.print_helm_diff(stdout)
-        except Exception as exception:
-            if len(exception.args) == 1 and "release: not found" in exception.args[0]:
```

Comment:
What happened to this?

Reply:
The helm wrapper already catches the exception and exits if something doesn't work properly. I thought this might be a bit redundant and too generic for exception handling because we are catching `Excepition`.